packages/staking-contracts/src/safe-transaction-proposer.ts

```diff
@@ -128,11 +128,8 @@ export class SafeTransactionProposer implements RewardsAllocator {
       senderSignature: signature.data,
     });
 
-    // NOTE: There's no way to know ahead of the time the final tx hash, so we return a placeholder
```

Comment:
The `SafeTransactionProposer` returns `null` now and not the zero hash. That means when we allocate using `SafeTransactionProposer` tx will be empty